src/main/java/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -1,32 +1,129 @@
 package org.usfirst.frc4904.robot;
 
+import org.usfirst.frc4904.standard.custom.controllers.CustomJoystick;
 import org.usfirst.frc4904.standard.custom.controllers.CustomXbox;
+import org.usfirst.frc4904.robot.humaninterface.HumanInterfaceConfig;
+import org.usfirst.frc4904.standard.subsystems.chassis.TankDriveShifting;
+import org.usfirst.frc4904.standard.subsystems.chassis.SolenoidShifters;
+import org.usfirst.frc4904.standard.subsystems.motor.Motor;
+import org.usfirst.frc4904.standard.subsystems.motor.speedmodifiers.EnableableModifier;
+import org.usfirst.frc4904.standard.subsystems.motor.speedmodifiers.AccelerationCap;
+import org.usfirst.frc4904.standard.custom.sensors.PDP;
+import org.usfirst.frc4904.standard.custom.PCMPort;
+import org.usfirst.frc4904.standard.custom.sensors.CANEncoder;
+import org.usfirst.frc4904.standard.custom.sensors.EncoderPair;
+import org.usfirst.frc4904.standard.custom.motioncontrollers.CustomPIDController;
+import org.usfirst.frc4904.standard.custom.sensors.NavX;
+import org.usfirst.frc4904.robot.subsystems.Climber;
+import edu.wpi.first.wpilibj.VictorSP;
 
 
 public class RobotMap {
     public static class Port {
         public static class HumanInput {
             public static final int xboxController = 1;
-
+            public static final int joystick = 0; //Change port later
+        }
+        public static class CANMotor {}  
+        public static class PWM {
+            public static final int leftDriveA = 2; // Not real ports, change these
+			public static final int leftDriveB = 3;
+			public static final int rightDriveA = 0;
+			public static final int rightDriveB = 1;
+        }
+        public static class CAN {
+            public static final int leftEncoder = -1; // Not real ports, change these
+			public static final int rightEncoder = -1;
+        }
+        public static class Pneumatics {
+            public static final PCMPort shifter = new PCMPort(1, 0, 1);
+            public static final PCMPort climberFrontLift = new PCMPort(-1, -1, -1); //Define Pneumatics.prongIOPusher with a correct port.
+            public static final PCMPort climberBackLift = new PCMPort(-1, -1, -1); //Define Pneumatics.prongIOPusher with a correct port.
         }
-        public static class CANMotor {}
-        public static class PWM {}
-        public static class CAN {}
-        public static class Pneumatics {}
     }
     
-    public static class Metrics{}
+    public static class Metrics{
+        public static class Wheel {
+			public static final double TICKS_PER_REVOLUTION = 4.904; // Obviously not real metrics
+			public static final double DIAMETER_INCHES = 4.904;
+			public static final double CIRCUMFERENCE_INCHES = Metrics.Wheel.DIAMETER_INCHES * Math.PI;
+			public static final double TICKS_PER_INCH = Metrics.Wheel.TICKS_PER_REVOLUTION
+				/ Metrics.Wheel.CIRCUMFERENCE_INCHES;
+			public static final double DISTANCE_FRONT_BACK = 4.904;
```

Comment:
Set every 4.904 to -1. The only reason we have 49.04 in the code last year was because that was the actual length of the robot.